### DISCUSSION TITLE
希望稍后再看和播放列表内的视频显示视频时长

### DISCUSSION THREAD

**Discussion by falling7down** [ywmoyue/biliuwp-lite#752] [feature]
### 问题描述

播放列表内的视频标题下面空了一行

### 描述解决方案

可以在播放列表内的封面内显示视频时长，视频标题下面增加上传时间，播放量弹幕等信息；
稍后再看封面里也增加视频时长显示，历史记录增加播放进度，如已看完，01:25/3:55等等

### 代码贡献

否

**Comment by ywmoyue** [comment]
播放列表中不好加，在播放列表中不一定能拿到对应的视频信息
